web_src/js/components/RepoContributors.vue

```diff
@@ -0,0 +1,353 @@
+<template>
+  <div>
+    <h2 class="ui header">
+      <relative-time
+        v-if="dateFrom !== null"
+        format="datetime"
+        year="numeric"
+        month="short"
+        day="numeric"
+        weekday=""
+        :datetime="dateFrom"
+      >
+        {{ dateFrom }}
+      </relative-time>
+      {{ isLoading ? "Loading contributions..." : errorText ? "Loading contributions failed :(": "-" }}
```

Comment:
Need to use text from the translation system. Add entries to `options/locale/locale_en-US.ini` and then pass them down to Vue via HTML attributes, see other Vue components for examples.

Reply:
@silverwind  What if I need translations with numbers? (1 commit, 3 commits etc.) I couldn't find any example for this.